cmd/completion.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// getCmd represents the get command
+func newCompletionCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "completion [bash|zsh|fish]",
+		Short: "generate completion script",
```

Comment:
can you add `SilenceUsage: true` under the Command to avoid printing all the help in case or error ?

Reply:
I've amended the commit and added the option.